Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
+++ .idea/vcs.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/build.gradle	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/build.gradle	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -34,6 +34,7 @@
 }
 
 dependencies {
+    implementation 'androidx.browser:browser:1.2.0'
     implementation 'com.google.firebase:firebase-bom:28.0.1'
     implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation 'com.google.android.material:material:1.3.0'
Index: app/proguard-rules.pro
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/proguard-rules.pro	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/proguard-rules.pro	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -18,4 +18,7 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+-keepattributes Signature
+-keepattributes *Annotation*
\ No newline at end of file
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/AndroidManifest.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/AndroidManifest.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -3,6 +3,7 @@
     package="com.Health.health">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.SEND_SMS" />
 
     <application
         android:allowBackup="true"
@@ -12,7 +13,7 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.Health">
         <activity android:name=".trainertab"></activity>
-        <activity android:name=".Centersignup" />
+        <activity android:name=".auth" />
         <activity android:name=".Login">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
Index: app/src/main/java/com/Health/health/Centersignup.java
===================================================================
--- app/src/main/java/com/Health/health/Centersignup.java	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/java/com/Health/health/Centersignup.java	(revision deae94ae83ac93f787d5868ea2f707c269353068)
@@ -1,93 +0,0 @@
-package com.Health.health;
-
-import androidx.annotation.NonNull;
-import androidx.appcompat.app.AppCompatActivity;
-
-import android.content.Intent;
-import android.os.Bundle;
-import android.text.Editable;
-import android.text.TextWatcher;
-import android.util.Log;
-import android.view.KeyEvent;
-import android.view.View;
-import android.widget.Button;
-import android.widget.EditText;
-import android.widget.TextView;
-import android.widget.Toast;
-
-import com.google.android.gms.tasks.OnCompleteListener;
-import com.google.android.gms.tasks.Task;
-import com.google.firebase.database.DataSnapshot;
-import com.google.firebase.database.DatabaseError;
-import com.google.firebase.database.DatabaseReference;
-import com.google.firebase.database.FirebaseDatabase;
-import com.google.firebase.database.ValueEventListener;
-
-import java.util.ArrayList;
-
-public class Centersignup extends AppCompatActivity {
-    EditText code;
-    Button next;
-    String code_value;
-    private FirebaseDatabase firebaseDatabase;
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_centersignup);
-
-        code=(EditText)findViewById(R.id.center_code);
-        code.addTextChangedListener(new TextWatcher() {
-            @Override
-            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
-
-            }
-
-            @Override
-            public void onTextChanged(CharSequence s, int start, int before, int count) {
-
-            }
-
-            @Override
-            public void afterTextChanged(Editable s) {
-                code_value=s.toString();
-                Log.e("code_value",code_value);
-
-            }
-        });
-
-
-        next=(Button)findViewById(R.id.register);
-        next.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-
-                FirebaseDatabase database = FirebaseDatabase.getInstance();
-                DatabaseReference reference = database.getReference().child("fitness");
-                reference.addListenerForSingleValueEvent(new ValueEventListener() {
-                    @Override
-                    public void onDataChange(@NonNull DataSnapshot snapshot) {
-                        for (DataSnapshot value : snapshot.getChildren()){
-                            if(code_value.equals(value.getValue().toString())){
-                                Intent intent = new Intent(Centersignup.this,membersignup.class);
-                                intent.putExtra("account_type",1);
-                                startActivity(intent);
-                            }
-                            else{
-                                Toast.makeText(Centersignup.this,"가입되지 않은 센터입니다.",Toast.LENGTH_SHORT).show();
-                                return;
-                            }
-                            Log.e("value.getvalue", String.valueOf(value.getValue()));
-
-                        }
-                    }
-
-                    @Override
-                    public void onCancelled(@NonNull DatabaseError error) {
-                        Log.e("example_1", String.valueOf(error));
-                    }
-                });
-            }
-        });
-
-    }
-}
\ No newline at end of file
Index: app/src/main/java/com/Health/health/Login.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/Health/health/Login.java	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/java/com/Health/health/Login.java	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -37,18 +37,11 @@
         signup.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Intent intent = new Intent(Login.this, membersignup.class);
-                startActivity(intent);
-            }
-        });
-
-        Centersignup.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Intent intent = new Intent(Login.this,Centersignup.class);
+                Intent intent = new Intent(Login.this, auth.class);
                 startActivity(intent);
             }
         });
+
         signin.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
Index: app/src/main/java/com/Health/health/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/Health/health/MainActivity.java	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/java/com/Health/health/MainActivity.java	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -42,11 +42,11 @@
                             Intent intent = new Intent(MainActivity.this,membertab.class);
                             startActivity(intent);
                         }
-                        else{
+                        if(currentuser.equals(snapshot.child("trainer").child(currentuser).getKey())){
                             Intent intent = new Intent(MainActivity.this,trainertab.class);
                             startActivity(intent);
                         }
-                        Log.e("check_type",snapshot.child("Users").child(currentuser).getKey());
+                        Log.e("check_type", String.valueOf(snapshot.child("Users").child(currentuser).getValue()));
                     }
 
                     @Override
Index: app/src/main/java/com/Health/health/auth.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/Health/health/auth.java	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
+++ app/src/main/java/com/Health/health/auth.java	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -0,0 +1,221 @@
+package com.Health.health;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.content.IntentSender;
+import android.os.Bundle;
+import android.telephony.SmsManager;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.FirebaseException;
+import com.google.firebase.FirebaseTooManyRequestsException;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthInvalidCredentialsException;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.auth.PhoneAuthCredential;
+import com.google.firebase.auth.PhoneAuthOptions;
+import com.google.firebase.auth.PhoneAuthProvider;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.concurrent.TimeUnit;
+
+public class auth extends AppCompatActivity {
+    EditText code,phone;
+    Button next,auth_c,send;
+    String code_value,mPhone,replace_Phone;
+
+    private static final String TAG = "PhoneAuthActivity";
+
+    // [START declare_auth]
+    private FirebaseAuth mAuth;
+    // [END declare_auth]
+
+    private String mVerificationId;
+    private PhoneAuthProvider.ForceResendingToken mResendToken;
+    private PhoneAuthProvider.OnVerificationStateChangedCallbacks mCallbacks;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_auth);
+
+        mCallbacks = new PhoneAuthProvider.OnVerificationStateChangedCallbacks() {
+
+            @Override
+            public void onVerificationCompleted(PhoneAuthCredential credential) {
+                // This callback will be invoked in two situations:
+                // 1 - Instant verification. In some cases the phone number can be instantly
+                //     verified without needing to send or enter a verification code.
+                // 2 - Auto-retrieval. On some devices Google Play services can automatically
+                //     detect the incoming verification SMS and perform verification without
+                //     user action.
+                credential.getSmsCode();
+                Log.e(TAG, "onVerificationCompleted:" + credential.getSmsCode());
+
+                signInWithPhoneAuthCredential(credential);
+            }
+
+            @Override
+            public void onVerificationFailed(FirebaseException e) {
+                // This callback is invoked in an invalid request for verification is made,
+                // for instance if the the phone number format is not valid.
+                Log.e(TAG, "onVerificationFailed", e);
+
+                if (e instanceof FirebaseAuthInvalidCredentialsException) {
+                    // Invalid request
+                } else if (e instanceof FirebaseTooManyRequestsException) {
+                    // The SMS quota for the project has been exceeded
+                }
+
+                // Show a message and update the UI
+            }
+
+            @Override
+            public void onCodeSent(@NonNull String verificationId,
+                                   @NonNull PhoneAuthProvider.ForceResendingToken token) {
+                // The SMS verification code has been sent to the provided phone number, we
+                // now need to ask the user to enter the code and then construct a credential
+                // by combining the code with a verification ID.
+                // Save verification ID and resending token so we can use them later
+                mVerificationId = verificationId;
+                mResendToken = token;
+            }
+        };
+
+
+
+        next=(Button)findViewById(R.id.register);
+        mAuth = FirebaseAuth.getInstance();
+        send = (Button)findViewById(R.id.send);
+        auth_c = (Button)findViewById(R.id.auth_check);
+        phone=(EditText)findViewById(R.id.phone);
+        code=(EditText)findViewById(R.id.center_code);
+
+
+        phone.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+                mPhone = phone.getText().toString();
+                Log.e("phone",mPhone);
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable s) {
+
+
+            }
+        });
+
+
+
+        send.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                replace_Phone = mPhone.replaceFirst("[0]","+82");
+                Log.e("replace_phone",mPhone);
+                PhoneAuthOptions options = PhoneAuthOptions.newBuilder(mAuth.getInstance()).setPhoneNumber(replace_Phone)
+                        .setTimeout(60L,TimeUnit.SECONDS)
+                        .setActivity(auth.this)
+                        .setCallbacks(mCallbacks)
+                        .build();
+
+                PhoneAuthProvider.verifyPhoneNumber(options);
+                Log.e(TAG, String.valueOf(mCallbacks));
+            }
+        });
+        code.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable s) {
+                code_value=s.toString();
+                Log.e("code_value",code_value);
+
+
+
+            }
+        });
+        next.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                FirebaseDatabase database = FirebaseDatabase.getInstance();
+                DatabaseReference reference = database.getReference().child("fitness");
+                reference.addListenerForSingleValueEvent(new ValueEventListener() {
+                    @Override
+                    public void onDataChange(@NonNull DataSnapshot snapshot) {
+                        for (DataSnapshot value : snapshot.getChildren()){
+                            if(code_value.equals(value.getValue().toString())){
+                                Intent intent = new Intent(auth.this,membersignup.class);
+                                intent.putExtra("account_type",1);
+                                startActivity(intent);
+                            }
+                            else{
+                                Toast.makeText(auth.this,"가입되지 않은 센터입니다.",Toast.LENGTH_SHORT).show();
+                                return;
+                            }
+                            Log.e("value.getvalue", String.valueOf(value.getValue()));
+
+                        }
+                    }
+
+                    @Override
+                    public void onCancelled(@NonNull DatabaseError error) {
+                        Log.e("example_1", String.valueOf(error));
+                    }
+                });
+            }
+        });
+
+
+    }
+    private void signInWithPhoneAuthCredential(PhoneAuthCredential credential) {
+        mAuth.signInWithCredential(credential)
+                .addOnCompleteListener(this, new OnCompleteListener<AuthResult>() {
+                    @Override
+                    public void onComplete(@NonNull Task<AuthResult> task) {
+                        if (task.isSuccessful()) {
+                            // Sign in success, update UI with the signed-in user's information
+                            Log.d(TAG, "signInWithCredential:success");
+
+                            FirebaseUser user = task.getResult().getUser();
+                            // Update UI
+                        } else {
+                            // Sign in failed, display a message and update the UI
+                            Log.w(TAG, "signInWithCredential:failure", task.getException());
+                            if (task.getException() instanceof FirebaseAuthInvalidCredentialsException) {
+                                // The verification code entered was invalid
+                            }
+                        }
+                    }
+                });
+    }
+}
Index: app/src/main/java/com/Health/health/membersignup.java
===================================================================
--- app/src/main/java/com/Health/health/membersignup.java	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/java/com/Health/health/membersignup.java	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -18,10 +18,14 @@
 import com.google.firebase.auth.AuthResult;
 import com.google.firebase.auth.FirebaseAuth;
 import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.auth.PhoneAuthCredential;
+import com.google.firebase.auth.PhoneAuthOptions;
+import com.google.firebase.auth.PhoneAuthProvider;
 import com.google.firebase.database.DatabaseReference;
 import com.google.firebase.database.FirebaseDatabase;
 
 import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
 
 public class membersignup extends AppCompatActivity {
     private static final String TAG="RegisterActivity";
@@ -50,6 +54,7 @@
         mName = (EditText)findViewById(R.id.name);
         mregiser_btn = (Button)findViewById(R.id.register);
 
+
         mregiser_btn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
@@ -57,8 +62,14 @@
                 String pwd=mPasswordText.getText().toString().trim();
                 String pwdc=mPasswordcText.getText().toString().trim();
 
+                Log.e("email",email);
+                Log.e("pwd",pwd);
                 if(pwd.equals(pwdc)){
                     Log.d(TAG,"등록 버튼"+email+","+pwd);
+                    if (email.isEmpty()==true || pwd.isEmpty()==true){
+                        Toast.makeText(membersignup.this,"공백이 존재합니다.",Toast.LENGTH_SHORT).show();
+                        return;
+                    }
                     final ProgressDialog mDialog = new ProgressDialog(membersignup.this);
                     mDialog.setTitle("가입 진행중입니다.");
                     mDialog.show();
@@ -80,6 +91,7 @@
                                 hashMap.put("name",name);
                                 account_type=getIntent().getIntExtra("account_type",0);
 //                                String user_id=firebaseAuth.getCurrentUser().getUid();
+                                Log.e("account_type", String.valueOf(account_type));
                                 hashMap.put("account_type", String.valueOf(account_type));
                                 if (account_type==0){
                                     FirebaseDatabase database = FirebaseDatabase.getInstance();
@@ -108,12 +120,13 @@
                 }
                 else{
                     Toast.makeText(membersignup.this,"비밀번호가 틀렸습니다. 다시 입력해주세요.",Toast.LENGTH_SHORT).show();
-
                     return;
                 }
             }
         });
     }
+
+
     public boolean onSupportNavigatorUp(){
         onBackPressed();
         return super.onSupportNavigateUp();
Index: app/src/main/res/layout/activity_auth.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_auth.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
+++ app/src/main/res/layout/activity_auth.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".membersignup">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="100dp"
+        android:text="본인인증"
+        android:textAlignment="center"
+        android:textColor="@color/black"
+        android:textSize="50dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+    </TextView>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/phone_frame"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="60dp"
+        android:layout_marginTop="80dp"
+        android:layout_marginEnd="60dp"
+        android:orientation="vertical"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView"
+        app:passwordToggleEnabled="false"
+        android:baselineAligned="false">
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/phone"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:hint="Phone"
+            android:background="@color/white"
+            android:layout_gravity="center"
+            android:gravity="left"
+            android:inputType="numberDecimal">
+
+        </com.google.android.material.textfield.TextInputEditText>
+        <Button
+            android:id="@+id/send"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="인증번호 발송">
+
+        </Button>
+
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <LinearLayout
+        android:id="@+id/auth_frame"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="60dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="60dp"
+        android:orientation="horizontal"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/phone_frame"
+        >
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/auth_code"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:hint="인증번호를 입력해주세요."
+            style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+            >
+
+        </com.google.android.material.textfield.TextInputEditText>
+
+        <Button
+            android:id="@+id/auth_check"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="right"
+            android:text="인증하기"
+            android:gravity="center">
+
+        </Button>
+
+    </LinearLayout>
+
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/center_code_frame"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="60dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="60dp"
+        android:orientation="vertical"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/auth_frame"
+        app:passwordToggleEnabled="false"
+        android:baselineAligned="false">
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/center_code"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:hint="Center Code"
+            android:background="@color/white"
+            android:layout_gravity="center"
+            android:gravity="left">
+
+        </com.google.android.material.textfield.TextInputEditText>
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="\n귀하가 가입된 피트니스 센터의 고유번호를 입력해주세요.\n 개인 가입의 경우는 공백으로 놔두세요.\n"
+            android:textColor="@color/black"
+            android:textSize="10dp"
+            android:gravity="center">
+
+        </TextView>
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <Button
+        android:id="@+id/register"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="60dp"
+        android:layout_marginTop="50dp"
+        android:layout_marginEnd="60dp"
+        android:backgroundTint="@color/white"
+        android:text="진행하기"
+        android:textColor="@color/black"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/center_code_frame">
+
+    </Button>
+
+
+</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/res/layout/activity_centersignup.xml
===================================================================
--- app/src/main/res/layout/activity_centersignup.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/res/layout/activity_centersignup.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
@@ -1,66 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".membersignup">
-
-    <TextView
-        android:id="@+id/textView"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="100dp"
-        android:text="트레이너 회원가입"
-        android:textAlignment="center"
-        android:textColor="@color/black"
-        android:textSize="50dp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
-
-    </TextView>
-    <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/center_code_frame"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="60dp"
-        android:layout_marginTop="90dp"
-        android:layout_marginEnd="60dp"
-        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView"
-        app:passwordToggleEnabled="false"
-        android:baselineAligned="false">
-        <com.google.android.material.textfield.TextInputEditText
-            android:id="@+id/center_code"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:hint="Center Code"
-            android:background="@color/white"
-            android:layout_gravity="center"
-            android:gravity="left">
-
-        </com.google.android.material.textfield.TextInputEditText>
-    </com.google.android.material.textfield.TextInputLayout>
-
-
-    <Button
-        android:id="@+id/register"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="60dp"
-        android:layout_marginTop="50dp"
-        android:layout_marginEnd="60dp"
-        android:backgroundTint="@color/white"
-        android:text="진행하기"
-        android:textColor="@color/black"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/center_code_frame">
-
-    </Button>
-
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
Index: app/src/main/res/layout/activity_membersignup.xml
===================================================================
--- app/src/main/res/layout/activity_membersignup.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/res/layout/activity_membersignup.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -43,10 +43,9 @@
             android:gravity="left">
 
         </com.google.android.material.textfield.TextInputEditText>
-
-
     </com.google.android.material.textfield.TextInputLayout>
 
+
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/email_frame"
         android:layout_width="match_parent"
Index: app/src/main/res/values/colors.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/colors.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/res/values/colors.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -2,7 +2,7 @@
 <resources>
     <color name="purple_200">#FFBB86FC</color>
     <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
+    <color name="purple_700">#000080</color>
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
Index: app/src/main/res/values/themes.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/values/themes.xml	(revision deae94ae83ac93f787d5868ea2f707c269353068)
+++ app/src/main/res/values/themes.xml	(revision 35d93a051f61f15452bb9622ef4b6ebfa64a8430)
@@ -2,7 +2,7 @@
     <!-- Base application theme. -->
     <style name="Theme.Health" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_500</item>
+        <item name="colorPrimary">#000080</item>
         <item name="colorPrimaryVariant">@color/white</item>
         <item name="colorOnPrimary">@color/white</item>
         <!-- Secondary brand color. -->
